Makefile

```diff
@@ -84,6 +90,76 @@ else
 	@echo Update complete
 endif
 
+## zkgroup
+build-zkgroup:
+	@echo "Found cargo with version $(CARGO_VERSION)"
+	@echo "Found go with version $(GO_VERSION)"
+	@echo "Found go with version $(GIT_VERSION)"
+ifeq ($(UNAME_S), Linux)
+ifeq ($(UNAME_HARDWARE_PLATTFORM), x86_64)
+	@echo "get zkgroup $(PLATFORM)"
+	go get -d github.com/nanu-c/zkgroup
```

Comment:
In other places in the Makefile, whenever a binary is used, its "alias" is used which uses the full path to the binary.

For example $(NPM), $(FLATPAK) and $(SNAP).

For go, there is a similar setup prepared, which uses the $(GO) "alias". This is for example already in place for `build-axolotl`. What do you think about replacing the direct `go` usage here with the $(GO) direct path?